File: conn.php
<?php
$servername = "localhost";
$username = "root";
$password = "usbw";
$dbname = "loginsystem";

// Create connection
$conn = new mysqli($servername, $username, $password, $dbname);

// Check connection
if ($conn->connect_error) {
    die("Connection failed: " . $conn->connect_error);
} 
$name=mysqli_real_escape_string($conn,$_REQUEST['lsu_name']);
$email=mysqli_real_escape_string($conn,$_REQUEST['lsu_email']);
$password=mysqli_real_escape_string($conn,$_REQUEST['lsu_password']);

$sql="INSERT INTO ls_user (lsu_name, lsu_email, lsu_password) VALUES ('$name', '$email', '$password')";

if(mysqli_query($conn, $sql))
{
    echo "Register Sucessfully.";
}
else{
    echo "Error";
}

mysqli_close($conn);
?>

File: conn.py
#! D:\laragon\bin\python\python-3.6.1/python.exe
print("Content-Type: text/html")
print()

import cgi, cgitb
import mysql.connector
#import webbrowser

print ("<html>")
print ("<head>")
print ("<title>Python world</title>")
print ("</head>")
print ("<body>")
form = cgi.FieldStorage() 
name = form.getvalue("lsu_name")
email = form.getvalue("lsu_email")
password = form.getvalue("lsu_password")
print (name)
print (email)
print (password)
cnx = mysql.connector.connect(host="localhost", user="root", passwd="usbw", database="loginsystem")
if cnx:
    print("\r\n Register Successfully")
    mycursor = cnx.cursor()
    mycursor.execute("INSERT INTO ls_user (lsu_name, lsu_email, lsu_password) VALUES (%s, %s, %s)", (name, email, password))
    cnx.commit()
    print ("\r\n Record inserted successfully")    

print ('<a href="index.html">Login Here</a>')
print ("</body>")
print ("</html>")
#page =  urllib.urlopen('index.html').read()
#print (page)